doc/stages/filters.outlier.rst

```diff
@@ -47,7 +47,7 @@ A global mean :math:`\overline{\mu}` of these mean distances is then computed al
 
 where :math:`m` is a user-defined multiplier specified by multiplier_.
 
-We now interate over the pre-computed mean distances :math:`\mu_i` and compare to computed threshold value. If :math:`\mu_i` is greater than the threshold, it is marked as an outlier.
+We now integrate over the pre-computed mean distances :math:`\mu_i` and compare to computed threshold value. If :math:`\mu_i` is greater than the threshold, it is marked as an outlier.
```

Comment:
Nope, should've been `iterate` ;)

Reply:
... but, whatever the word, it's spelled correctly now 😄